### PR TITLE
Little proxy

### PR DESCRIPTION
**Pull request by ludwikkazmierczak**
- Changed default proxy implementation used from jetty based to little proxy
- Added configuration to kill idle requests after 90 seconds
- Firefox to use Marionette
- Moved `TestAdsEvolveMercury`  and  `TestAdsScreenshotComparisonMobile` to use Nexus 5 emulator

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/common/core/drivers/browsers/FirefoxBrowser.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import org.openqa.selenium.WebDriverException;
 import org.openqa.selenium.firefox.FirefoxDriver;
 import org.openqa.selenium.firefox.FirefoxProfile;
+import org.openqa.selenium.firefox.MarionetteDriver;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/drivers/browsers/FirefoxBrowser.java]: MINOR Remove this unused import 'org.openqa.selenium.firefox.MarionetteDriver'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsEvolveMercury.java [hunk 2]
```diff
@@ -1,15 +1,20 @@
 package com.wikia.webdriver.testcases.adstests;
 
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
 import com.wikia.webdriver.common.core.geoedge.CountryCode;
 import com.wikia.webdriver.common.core.geoedge.GeoEdgeBrowserMobProxy;
+import com.wikia.webdriver.common.core.helpers.Emulator;
 import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
 import com.wikia.webdriver.common.templates.mobile.MobileTestTemplate;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsEvolveMercury.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.templates.mobile.MobileTestTemplate'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/common/core/networktrafficinterceptor/NetworkTrafficInterceptor.java [hunk 3]
```diff
@@ -1,40 +1,30 @@
 package com.wikia.webdriver.common.core.networktrafficinterceptor;
 
-import java.util.HashMap;
-import java.util.Map;
+import java.net.InetSocketAddress;
+import java.util.concurrent.TimeUnit;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/networktrafficinterceptor/NetworkTrafficInterceptor.java]: MINOR Remove this unused import 'java.util.concurrent.TimeUnit'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/common/core/networktrafficinterceptor/NetworkTrafficInterceptor.java [hunk 4]
```diff
@@ -1,40 +1,30 @@
...
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import net.lightbody.bmp.BrowserMobProxyServer;
+import net.lightbody.bmp.client.ClientUtil;
 import net.lightbody.bmp.core.har.Har;
 import net.lightbody.bmp.core.har.HarEntry;
-import net.lightbody.bmp.proxy.ProxyServer;
+import net.lightbody.bmp.proxy.CaptureType;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/networktrafficinterceptor/NetworkTrafficInterceptor.java]: MINOR Remove this unused import 'net.lightbody.bmp.proxy.CaptureType'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsScreenshotComparisonMobile.java [hunk 5]
```diff
@@ -1,16 +1,21 @@
...
 
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
 import com.wikia.webdriver.common.core.geoedge.CountryCode;
 import com.wikia.webdriver.common.core.geoedge.GeoEdgeBrowserMobProxy;
+import com.wikia.webdriver.common.core.helpers.Emulator;
 import com.wikia.webdriver.common.core.url.UrlBuilder;
 import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
 import com.wikia.webdriver.common.templates.mobile.MobileTestTemplate;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsScreenshotComparisonMobile.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.templates.mobile.MobileTestTemplate'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues:
* MINOR 5 minor

Watch the comments in this conversation to review them.

**Comment by jacekjursza** [comment]
LGTM 👍

**Pull request merged**